ocs_ci/ocs/cluster.py

```diff
@@ -632,6 +632,34 @@ def time_taken_to_complete_rebalance(self, timeout=600):
                 time_taken = time.time() - start_time
                 return (time_taken / 60)
 
+    def get_clustername(self):
+        """
+        Return the name (DNS short name) of the cluster
+
+        Args:
```

Comment:
no need to mention args if there are none

Reply:
removed.